src/content/capture/canvas-pc/css/main.css

```diff
@@ -8,36 +8,17 @@
 
 canvas {
   background-color: #ccc;
-  height: 231px;
-  margin: 0 20px 0 0;
+  --width: calc(45%);
+  width: var(--width);
+  height: calc(var(--width) * 0.75);
+  margin: 1em;
   vertical-align: top;
-  width: calc(50% - 12px);
 }
 
 video {
-  height: 231px;
-  margin: 0 0 20px 0;
+  --width: calc(45%);
```

Comment:
If you want to define the width only once, you could put the variable definition in the `html` block (though that might be less clear!)

Reply:
I was considering that at first, but the value would be different in the samples. This makes it a bit redundant, but at least the samples are more isolated.